chapters/classes.tex

```diff
@@ -1511,6 +1511,8 @@ \subsection{Real Type}\label{real-type}
 
 The following attributes shall be evaluable: \lstinline!quantity!, \lstinline!unit!, \lstinline!displayUnit!, \lstinline!fixed!, and \lstinline!stateSelect!.
 
+The \lstinline!quantity!-attribute is used to impose a constraint on connection sets, see \cref{connect-set-quantity-rule}.
```

Comment:
I believe this should be after the unit and displayUnit-attributes, as one reason to have quantity is that variables may have the same unit, but still be different - and we use quantity to differentiate them. 
We could in this sub-section add "It is intended to be more fine-grained than the unit."; which makes it clearer that it should be after the unit to avoid a forward reference.

Reply:
I was just following the order in the record, but with a change of order in the record I see no problem with also swapping the explaining paragraphs.

I am hesitant to add
> It is intended to be more fine-grained than the unit.

since I have seen (rare) examples where it is not the case, where incompatible units are used with the same quantity.  Also, since both `"m"` and `"mm"` could be used with `quantity = "Length"`, it rather seems as if the unit would be more fine-grained than the quantity.

That said, I see your point if those rare examples are ignored and units are thought of as equivalence classes, so maybe we can find something similar that is more safe to say…  I was thinking for a while, but couldn't come up with anything brief that would be meaningful to add here.  I suggest we leave this for a separate PR.